openhtf/exe/phase_executor.py

```diff
@@ -55,6 +57,16 @@
 class InvalidPhaseResultError(Exception):
   """Raised when a PhaseOutcome is created with an invalid phase result."""
 
+class ExceptionOutcome(collections.namedtuple('ExceptionOutcome', ['exc_type','exc_val','exc_tb'])):
```

Comment:
Can we name this `ExceptionInfo` to match the python `exc_info` nomenclature?  Also, line too long :)